content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Configuring Gateway Network Topology (Development)
+description: How to configure gateway network topology (Development).
+weight: 60
+keywords: [traffic-management,ingress,gateway]
+---
+
+## Configuring network topologies (Development)
+
+{{< idea >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{</ idea >}}
+
+Istio provides the ability to manage settings like [X-Forwarded-For](https://tools.ietf.org/html/rfc7239)
+([XFF](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for)
```

Comment:
The first link doesn't seem right, the link to the envoy x-forwarded-for is inconsistent with the x-forwarded-client-cert, and there is a missing close paren `)` in the sentence. Is this what you want?
```suggestion
Istio provides the ability to manage settings like [X-Forwarded-For](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for) (XFF)
```

Reply:
I updated it and will have Adam review it. Thanks.